dolphinscheduler-service/src/test/java/org/apache/dolphinscheduler/service/log/LogClientTest.java

```diff
@@ -151,8 +151,7 @@ public void testRemoveTaskLog() throws Exception {
                     .thenReturn(command);
 
             LogClient logClient = new LogClient();
-            Boolean status = logClient.removeTaskLog(Host.of("localhost:1234"), "/log/path");
-            Assertions.assertTrue(status);
+            Assertions.assertDoesNotThrow(() -> logClient.removeTaskLog(Host.of("localhost:1234"), "/log/path"));
```

Comment:
Because you have a `catch (Exception e)` in `logClient.removeTaskLog` method, the `removeTaskLog` won't throw exception anyhow, so there is no point to assert it does not throw.

Reply:
Yes, this assertion is only want to make sure this method will never throw exception.